nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw add dev list section title

# Description
Centered the We Are... Animated in the DevList; added the subheading to DevList.

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions
- Fetch branch and run site in browser.
- You should see a h2 subheading above the Developer List of students
- You should see a text animation just below the subheading list

# Checklist:
